modules/parsers/parser-hdbschema/com.sap.xsk.parser.hdbschema/src/main/antlr4/com/sap/xsk/parser/hdbschema/core/Hdbschema.g4

```diff
@@ -0,0 +1,13 @@
+grammar Hdbschema;
+
+hdbschemaDefinition: property+;
```

Comment:
1. According to the documentation: https://help.sap.com/viewer/cc2b23beaa3344aebffa2f6e717df049/1.0.12/en-US/6cdea002ce634a8c81181de92bc0d8ea.html
only one schema should be specify, there for  no need for 'property+'.

```
hdbschemaDefinition: schemaProp;
schemaProp: 'schema_name' EQ STRING SEMICOLON;
```


2. The definition should cover the 
schema_name="MYSCHEMA";
not 
schema="MYSCHEMA";